Parser/asdl_c.py

```diff
@@ -1148,10 +1146,7 @@ def simpleSum(self, sum, name):
             self.emit("Py_INCREF(astmodulestate_global->%s_singleton);" % t.name, 3)
             self.emit("return astmodulestate_global->%s_singleton;" % t.name, 3)
         self.emit("default:", 2)
-        self.emit('/* should never happen, but just in case ... */', 3)
-        code = "PyErr_Format(PyExc_SystemError, \"unknown %s found\");" % name
-        self.emit(code, 3, reflow=False)
-        self.emit("return NULL;", 3)
+        self.emit("Py_UNREACHABLE();", 3);
```

Comment:
I would remove the `default` cause and add `Py_UNREACHABLE()` after the `switch` statement. The advantage is that the compiler may warn if `switch` does not have `case`s for all enum values and there is no `default`.

Reply:
Didn't know about it. Thanks.